src/Admin/routes.php

```diff
@@ -16,4 +17,9 @@
         'index',
         $route->toAdmin()
     );
+    $map->post(
+        '/testmail',
```

Comment:
My OCD would prefer `/mail/test`, to be honest. :grin: 

Maybe we should change the settings endpoint to `/mail/settings` then? :thinking:

Reply:
Oh wait, this isn't the API? Interesting. You *are* returning JSON here, but it's not a typical JSON:API endpoint.

Reply:
I'm going to move this to the API, it doesn't make sense to have JSON Endpoints in admin.